polynote-frontend/polynote/data/messages.ts

```diff
@@ -61,19 +61,6 @@ export class LoadNotebook extends Message {
     }
 }
 
-export class CloseNotebook extends Message {
```

Comment:
This message is no longer necessary – to close the notebook we just close the websocket.

Reply:
With this approach can we differentiate between a purposefully closed websocket and one closed due to other reasons (network interrupt, etc)? (or maybe we don't need to care about differentiating)